builder-images/hacbs-jdk11-builder/Dockerfile

```diff
@@ -0,0 +1,56 @@
+FROM registry.access.redhat.com/ubi8/openjdk-11
+USER root
+ENV \
+    LANG="en_US.UTF-8" \
+    LANGUAGE="en_US:en" \
+    LC_ALL="en_US.UTF-8" \
+    APP_HOME="/home/hacbs" \
+    MAVEN_HOME="/usr/share/maven" \
+    MAVEN_OPTS="-XX:+TieredCompilation -XX:TieredStopAtLevel=1" \
+    MAVEN_VERSION="3.8.6" \
+    GRADLE_HOME="/usr/share/gradle" \
+    GRADLE_OPTS="-Dorg.gradle.daemon=false" \
+    GRADLE_VERSION="7.5" \
+    HOME="/home/hacbs" \
+    NSS_WRAPPER_PASSWD="/etc/passwd" \
+    GRADLE_DOWNLOAD_SHA256=cb87f222c5585bd46838ad4db78463a5c5f3d336e5e2b98dc7c0c586527351c2 \
+    PATH="$PATH:$JAVA_HOME/bin" \
+    BASE_URL=https://apache.osuosl.org/maven/maven-3/3.8.6/binaries \
+    MAVEN_SHA=f790857f3b1f90ae8d16281f902c689e4f136ebe584aba45e4b1fa66c80cba826d3e0e52fdd04ed44b4c66f6d3fe3584a057c26dfcac544a60b301e6d0f91c26
+
+RUN groupadd -r hacbs -g 1001 && useradd -u 1001 -r -g 1001 -m -d ${APP_HOME} -s /sbin/nologin -c "HACBS user" hacbs && cp /etc/passwd /home/hacbs/passwd && chown 1001:1001 /home/hacbs/passwd && mkdir /project && chown 1001:1001 /project
+
+RUN microdnf --setopt=install_weak_deps=0 --setopt=tsflags=nodocs install -y apr-devel wget tar gzip gcc glibc-devel zlib-devel shadow-utils unzip gcc-c++ glibc-langpack-en buildah findutils podman bzip2-devel gcc-c++ libcurl-devel openssl-devel bc which lsof unzip git libstdc++-static glibc-static make autoconf automake libtool tar glibc-devel libgcc.i686 glibc-devel.i686\
+    && microdnf clean all \
+    && rpm -q apr-devel wget tar gzip gcc glibc-devel zlib-devel shadow-utils unzip gcc-c++ glibc-langpack-en buildah findutils podman bzip2-devel gcc-c++ libcurl-devel openssl-devel bc which lsof unzip git libstdc++-static glibc-static make autoconf automake libtool tar glibc-devel libgcc.i686 glibc-devel.i686
```

Comment:
curious why this verification is needed?

Reply:
This was copied/adapted from the CEKit generated dockerfiles. I just assumed this was best practice because CEKit does it.